java/lib/src/main/java/svix/Application.java

```diff
@@ -13,23 +13,23 @@
 		api = new ApplicationApi();
 	}
 
-	public ListResponseApplicationOut list(String iterator, Integer limit) throws ApiException {
+	public ListResponseApplicationOut list(final String iterator, final Integer limit) throws ApiException {
```

Comment:
As far as I know it's not common to set 'final' on method params
It's fine to keep it and doesn't really change a lot though

Reply:
Ah, I added Github's Superlinter who's default setup yelled about this, I can disable the rule and undo this change if you think this is just added noise.

Reply:
I think having `final` is probably good. It's maybe just not Google-style, but I don't think it detracts anything. Following what the linter says sounds good.